### PR TITLE
Update NLP & HuggingFace Demo

### PR DESCRIPTION
**Pull request by RachelXu7**

### CODE REVIEW

File: demo/auto_compression/nlp/README.md [hunk 1]
```diff
@@ -119,8 +119,7 @@ python run.py \
     --task_name='afqmc' \
     --config_path='./configs/pp-minilm/auto/afqmc.yaml'
 ```
-如仅需验证模型精度，在启动```run.py```脚本时，命令加上```--eval=True```即可。
-
+如仅需验证模型精度，或验证压缩之后模型精度，在启动```run.py```脚本时，将命令行中模型文件夹 ```model_dir``` 改为压缩之后保存的文件夹路径 ```./save_afqmc_pruned/``` ，命令加上```--eval=True```即可。
```
Comment (ceci3) [inline demo/auto_compression/nlp/README.md]: 把完整指令也加一下吧

File: demo/auto_compression/pytorch_huggingface/run.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 from functools import partial
 from paddle.io import Dataset, BatchSampler, DataLoader
 from paddle.metric import Metric, Accuracy
-from paddlenlp.transformers import BertForSequenceClassification, BertTokenizer
+from transformers import AutoTokenizer
```
Comment (ceci3) [inline demo/auto_compression/pytorch_huggingface/run.py:25]: 这里需要新增依赖吗？

Comment (RachelXu7) [reply demo/auto_compression/pytorch_huggingface/run.py:25]: 安装huggingface即可

Comment (ceci3) [reply demo/auto_compression/pytorch_huggingface/run.py:25]: 那把 huggingface 的安装也放在readme的依赖里？

**Review by ceci3 — APPROVED** [review]
